### ISSUE TITLE
Weighted loss function?

### ISSUE THREAD

**Issue by rnditdev** [google-research/deeplab2#53]
Hello All,
In order to segment unbalanced dataset, a class-weighted loss function is a must have requirements. Do you have plans to implement class-weighted loss functions? Any hint about how to go for implementing it by myself?
Best,

**Comment by aquariusjay** [comment]
Hi @rnditdev,

Thanks for the question.
Unfortunately, we currently do not have any plan to support class-weighted loss function.
However, you could refer to our [TensorFlow1 implementation](https://github.com/tensorflow/models/blob/238922e98dd0e8254b5c0921b241a1f5a151782f/research/deeplab/utils/train_utils.py#L51) for the support.
If you are interested in contributing to this, it is very welcome, and we will be happy to review the PR.

Cheers,

**Comment by aquariusjay** [comment]
Closing the issue since it is not active for a long time.
Please feel free to reopen it if needed.
